cas-client-core/src/main/java/org/jasig/cas/client/session/SingleSignOutHandler.java

```diff
@@ -172,7 +183,19 @@ private boolean isLogoutRequest(final HttpServletRequest request) {
         }
         return false;
     }
-    
+
+    private boolean pathMatches(HttpServletRequest request) {
```

Comment:
The method name does not match its behavior. Your checking for equality while `matches` implies pattern matching.

Reply:
Or, if you think it makes sense, I can switch the implementation to pattern matching.

(I don't like the method name `pathEquals()`; maybe `pathIsApplicable()`?  `pathIsIncluded()`? `pathChecksOutOK()`? Honestly, I can't find anything I like more than `pathMatches()`.)

Reply:
If you go with pattern matching, then the method name probably works out :)

Reply:
Yeah, but it's hard (for me) to imagine a scenario when pattern matching is really what someone wants. Seems like it's better to KISS for this.

Reply:
Maybe you can think of such a scenario?